appservice/src/IFilesClient.ts

```diff
@@ -4,9 +4,15 @@
 *--------------------------------------------------------------------------------------------*/
 
 import KuduClient from 'vscode-azurekudu';
+import { IHostKeys } from './SiteClient';
 
 export interface IFilesClient {
 
     fullName: string;
+    defaultHostUrl: string;
+    isFunctionApp: boolean;
+    id: string;
+    kuduUrl: string | undefined;
     getKuduClient(): Promise<KuduClient>;
+    listHostKeys(): Promise<IHostKeys>;
```

Comment:
This is only used by Functions, so I don't think that it should be on the interface itself.

Reply:
Made optional